### DISCUSSION TITLE
Intermittent test failure locally

### DISCUSSION THREAD

**Discussion by Santhosh858** [getgauge/taiko#2659] [Development]
Some of the random tests are failing intermittently in the local. On re-running the failed tests, the issues were resolved automatically without any changes. Does anyone know the reason behind intermittent failure? Has anyone experienced similar issues and fixed it permanently? Any leads would be appreciated.

Below are the sample failed tests locally and were resolved automatically on re-running the test

**First time occurrence of issue:**
![image](https://user-images.githubusercontent.com/118368767/202205002-b970dbcf-80ba-4f06-820f-7acd4cd132e0.png)

**Fixed automatically on re-running the test:**
![image](https://user-images.githubusercontent.com/118368767/202205076-19cb9363-e427-47e9-b9cf-c7f3ccac659d.png)

**Comment by zabil** [comment]
Any code you can share, even a sample about what taiko commands you are using?
